install.sh

```diff
@@ -180,107 +241,190 @@ else
+	instlog "Downloading the Sub8 repository"
+	cd $CATKIN_DIR/src
+	git clone -q https://github.com/uf-mil/Sub8.git
+	cd $CATKIN_DIR/src/Sub8
+	git remote rename origin upstream
+	instlog "Make sure you change your git origin to point to your own fork! (git remote add origin your_forks_url)"
+fi
 
-#===================================#
-# Navigator Dependency Installation #
-#===================================#
 
-instlog "Installing Navigator dependencies from the Ubuntu repositories"
-sudo apt-get install -qq libompl-dev
-sudo apt-get install -qq python-pygame
+#================================#
+# Common Dependency Installation #
+#================================#
 
-instlog "Installing Navigator ROS dependencies"
-sudo apt-get install -qq ros-indigo-roboteq-driver
-sudo apt-get install -qq ros-indigo-rosserial
-sudo apt-get install -qq ros-indigo-rosserial-python ros-indigo-rosserial-arduino
+instlog "Installing common dependencies from the Ubuntu repositories"
 
-instlog "Installing Navigator dependencies from source"
-rm -rf /tmp/pyode-build
-mkdir -p /tmp/pyode-build
-cd /tmp/pyode-build
-sudo apt-get build-dep -qq python-pyode
-sudo apt-get remove -qq python-pyode
-apt-get source --compile -qq python-pyode
-sudo dpkg -i python-pyode_*.deb
+# Utilities for building and package management
+sudo apt-get install -qq cmake binutils-dev python-pip
 
+# Common backend libraries
+sudo apt-get install -qq libboost-all-dev
+sudo apt-get install -qq python-dev python-scipy python-numpy python-serial
 
-#==============================#
-# Sub8 Dependency Installation #
-#==============================#
-
-instlog "Installing Sub8 dependencies from the Ubuntu repositories"
-sudo apt-get update -qq
-sudo apt-get install -qq cmake binutils-dev python-pip
-sudo apt-get install -qq python-scipy python-pygame python-numpy python-serial
-sudo apt-get install -qq python-twisted socat
-sudo apt-get install -qq libvtk5-dev python-vtk
-sudo apt-get install -qq tesseract-ocr
-sudo apt-get install -qq libboost-all-dev python-dev python-qt4-dev python-qt4-gl python-opengl freeglut3-dev libassimp-dev
+# Point clouds
 sudo apt-get install -qq libpcl-1.7-all libpcl-1.7-all-dev
+
+# Motion planning
 sudo apt-get install -qq libompl-dev
-sudo apt-get install -qq libusb-1.0-0-dev
 
-instlog "Installing Sub8 ROS dependencies"
-sudo apt-get install -qq ros-indigo-gazebo7-msgs ros-indigo-gazebo7-ros ros-indigo-gazebo7-plugins ros-indigo-gazebo7-ros-control
-sudo apt-get install -qq ros-indigo-control-toolbox ros-indigo-controller-manager ros-indigo-transmission-interface ros-indigo-joint-limits-interface ros-indigo-hardware-interface
-sudo apt-get install -qq ros-indigo-sophus
+# Visualization and graphical interfaces
+sudo apt-get install -qq libvtk5-dev python-vtk
+sudo apt-get install -qq python-qt4-dev python-qt4-gl
+sudo apt-get install -qq python-opengl freeglut3-dev libassimp-dev
+
+# Tools
+sudo apt-get install -qq git-lfs gitk
```

Comment:
Do you not have to run that git lfs setup script that they have on the [website](https://github.com/github/git-lfs/blob/master/INSTALLING.md)? 

`curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash`